vector/src/main/java/im/vector/app/features/home/room/detail/timeline/helper/MessageInformationDataFactory.kt

```diff
@@ -119,7 +119,8 @@ class MessageInformationDataFactory @Inject constructor(private val session: Ses
                 isFirstFromThisSender = isFirstFromThisSender,
                 isLastFromThisSender = isLastFromThisSender,
                 e2eDecoration = e2eDecoration,
-                sendStateDecoration = sendStateDecoration
+                sendStateDecoration = sendStateDecoration,
+                liveLocationShareSummaryData = getLiveLocationShareSummaryData(event)
```

Comment:
Not sure it should be part of this class

Reply:
I tried to follow the same practice as for other extra information of events. Do you think I should extract it into its own `LiveLocationShareSummaryData` and associated `LiveLocationShareSummaryDataFactory`?

Reply:
You could just add this as attribute of needed items. MessageInformationData should be used for data used by all items

Reply:
I have removed the data class from `MessageInformationData` and encapsulated it into the live location item factory since this is the only area where it is used. See [cbf8306](https://github.com/vector-im/element-android/pull/5989/commits/cbf8306c6fe90907078278689b518230cfead989)